### PR TITLE
SLI-1032 Let users reopen resolved issues

### PR DESCRIPTION
**Pull request by nquinquenel**

### CODE REVIEW

File: src/main/java/org/sonarlint/intellij/editor/SonarExternalAnnotator.java [hunk 1]
```diff
@@ -135,6 +137,7 @@ private static void addAnnotation(Project project, LiveFinding finding, TextRang
 
     if (finding instanceof LiveIssue) {
       intentionActions.add(new MarkAsResolvedAction((LiveIssue) finding));
+      intentionActions.add(new ReopenIssueAction((LiveIssue) finding));
```
Comment (damien-urruty-sonarsource) [inline src/main/java/org/sonarlint/intellij/editor/SonarExternalAnnotator.java:140]: Is this possible? Do we annotate issues that are marked resolved? I'm not sure we want to bring attention on them for users

**Review by damien-urruty-sonarsource — APPROVED** [review]
A few comments but LGTM!
I think you can also update the release notes

File: src/main/java/org/sonarlint/intellij/actions/MarkAsResolvedAction.kt [hunk 2]
```diff
@@ -170,9 +174,14 @@ class MarkAsResolvedAction(
     }
 
     override fun isEnabled(e: AnActionEvent, project: Project, status: AnalysisStatus): Boolean {
-        // always disabled for standalone mode
-        // the checks will be done inside the popup for connected mode
-        return serverConnection(project) != null
+        var issue: Issue? = e.getData(ISSUE_DATA_KEY)
```
[the remark this answers is not in the document]
Comment (nquinquenel) [reply src/main/java/org/sonarlint/intellij/actions/MarkAsResolvedAction.kt:176]: Good point

File: src/main/java/org/sonarlint/intellij/editor/SonarExternalAnnotator.java (hunk 1 above)
Comment (nquinquenel) [reply src/main/java/org/sonarlint/intellij/editor/SonarExternalAnnotator.java:140]: I'll confirm this with PM, in the meantime I'm removing all the annotation part, just having them visible in the list is enough probably

Comment (nquinquenel) [reply src/main/java/org/sonarlint/intellij/editor/SonarExternalAnnotator.java]: It should not be annotated

**Pull request merged**